src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -133,7 +134,8 @@ public sealed class BaseFeature : FullNodeFeature, INodeStats
             IConsensusManager consensusManager,
             IConsensusRuleEngine consensusRules,
             IPartialValidator partialValidator,
-            IBlockStore blockStore = null)
+            IBlockPuller blockPuller,
+            IBlockStore blockStore)
```

Comment:
Ivan please create an issue to move block store to Stratis.Bitcoin if this is going to become a mandatory component and we would just implement pruning instead of not using a feature we need to track this.

Reply:
@bokobza FYI

Reply:
https://github.com/stratisproject/StratisBitcoinFullNode/issues/1845

done